### PR TITLE
volumetric colortogray and graytocolor

### PR DESCRIPTION
**Pull request by 0x00b1**

### CODE REVIEW

File: cellprofiler/modules/colortogray.py [hunk 1]
```diff
@@ -34,175 +36,333 @@
...
-            <li><i>%(CH_HSV)s:</i>The HSV (hue, saturation, value) color space is based on more intuitive color characteristics as
-            tint, shade and tone. Choosing
-            this option will split the image into any of the hue, saturation, and value component images.</li>
-            <li><i>%(CH_CHANNELS)s:</i>This is a more complex model for images which involve more than three channels.</li>
-            </ul>""" % globals())
+                <li><em>Split</em>: Splits the three channels 
+                (red, green, blue) of a color image into three separate
+                 grayscale images.</li>
+                
+                <li><em>Combine</em>: Converts a color image to a grayscale
```
Comment (mcquin) [inline cellprofiler/modules/colortogray.py]: Converts a grayscale image into a color image

File: cellprofiler/modules/colortogray.py [hunk 2]
```diff
@@ -34,175 +36,333 @@
...
+        _rgb_or_channels_doc = """
+        Many images contain color channels other than red, green and blue. For 
+        instance, GIF and PNG formats can have an alpha channel that encodes 
+        transparency. TIF formats can have an arbitrary number of channels 
+        which represent pixel measurements made by different detectors, 
+        filters or lighting conditions. This setting provides three options to 
+        choose from:
+
+        <ul>
+            <li><em>RGB</em>: The RGB (red,green,blue) color space is the
```
Comment (mcquin) [inline cellprofiler/modules/colortogray.py]: `(red, green, blue)`

File: cellprofiler/modules/graytocolor.py [hunk 3]
```diff
@@ -375,23 +415,38 @@ def display(self, workspace, figure):
             subplots = (min(nsubplots + 1, 4), int(nsubplots / 4) + 1)
             subplot_indices = [(i % 4, int(i / 4)) for i in range(nsubplots)]
             color_subplot = (nsubplots % 4, int(nsubplots / 4))
+
         figure.set_subplots(subplots)
-        for i, (input_image_name, image_pixel_data) in \
-                enumerate(zip(input_image_names, images)):
+
+        import IPython
+        IPython.embed()
```
Comment (mcquin) [inline cellprofiler/modules/graytocolor.py]: Remove me!

**Comment by mcquin** [comment]
For some input data, the grayscale images used to compose the multichannel output are displayed in color:

screen shot 2017-09-02 at 12 03 27 pm


It would be nice if the display for GrayToColor showed the images which are used to compose the multichannel output in grayscale.

**Comment by mcquin** [comment]
ColorToGray with "Split" conversion method raises the following error on display:

screen shot 2017-09-02 at 12 07 18 pm

**Comment by AnneCarpenter** [comment]
LMK when this one is merged and I can update the help/documentation.

**Comment by mcquin** [comment]
@0x00b1 bump

**Comment by mcquin** [comment]
Closing: inactivity

**Pull request closed without merging**